reconcile/openshift_cluster_bots.py

```diff
@@ -159,6 +163,26 @@ def create_sa(
             "type": "kubernetes.io/service-account-token",
         },
     ])
+    if cluster_admin:
+        items.append({
+            "apiVersion": "rbac.authorization.k8s.io/v1",
+            "kind": "ClusterRoleBinding",
+            "metadata": {
+                "name": f"{namespace}-{sa}",
```

Comment:
happy to change this if someone has a better idea...

Reply:
it's backwards compatible with what we have right now, so should be ok